reference/architecture/index.md

```diff
@@ -92,6 +94,14 @@ service.
  gate --> echo;
  gate --> igor(Igor);
  igor(Igor) --> echo;
+ 
+ keel(Keel) --> clouddriver;
+ keel --> orca;
+ keel --> front50;
+ keel --> fiat;
+ keel --> echo;
+ keel --> igor;
+ gate --> keel;
```

Comment:
This makes the diagram render like so:
![image](https://user-images.githubusercontent.com/1323478/103592621-5c99fd00-4ea8-11eb-8653-e58c4c1faffd.png)

It's not super pretty, but I found it hard to control the layout.